extensions/kubectl-cli/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Logs
```

Comment:
AFAIK this should be reduced

there is a global .gitignore at the root folder that include many of the ignore

this file also contains some reference to old tools that we don't use (bower, grunt), etc.

Reply:
That is default generated by github.com, will clean it up.

Reply:
It is obviously not needed at all, because one in the root of the repo works fine

Reply:
removed.